diesel/src/migrations/mod.rs

```diff
@@ -178,7 +179,7 @@ fn run_migrations<T, Conn: Connection>(conn: &Conn, migrations: T)
     -> Result<(), RunMigrationsError> where
         T: Iterator<Item=Box<Migration>>
 {
-    for migration in migrations {
+    for migration in migrations.into_iter().sorted_by(|a, b| Ord::cmp(&b.version(), &a.version())).into_iter().rev() {
```

Comment:
It would maybe be the better solution to implement Ord for Migration, but I was not able to do this.
As Far as I understood it's not possible to:
* Implement defaults for parent Traits functions
* Implement Traits for generic types in other modules as they are defined